content/en/docs/concepts/storage/volumes.md

```diff
@@ -338,6 +338,8 @@ simultaneous writers allowed.
 Using a PD on a Pod controlled by a ReplicationController will fail unless
 the PD is read-only or the replica count is 0 or 1.
 
+Support for [Regional Persistent Disks](https://cloud.google.com/compute/docs/disks/#repds) was introduced to Kubernetes v1.8 as alpha and moved to beta in Kubernetes v1.10. This feature allows the creation of Persistent Disks that are available in two zones within the same region. In order to use this feature, the volume must be provisioned as a PersistentVolume; referencing the volume directly from a pod is not supported.
```

Comment:
You can use this tag to denote a beta feature:
```
{{< feature-state for_k8s_version="v1.10" state="beta" >}}
```

Reply:
I don't think we need to mention 1.8 alpha because the storageclass parameters were also different.